src/SFA.DAS.EAS.Web/Validators/ChangeStatusViewModelValidator.cs

```diff
@@ -22,9 +22,16 @@ public ChangeStatusViewModelValidator(ICurrentDateTime currentDateTime)
                 {
                     RuleFor(r => r.DateOfChange)
                                .Cascade(CascadeMode.StopOnFirstFailure)
-                               .Must(d => d.DateTime.HasValue).WithMessage("Enter the stop date for this apprenticeship")
+                               .Must(d => d.DateTime.HasValue).WithMessage("Enter the stop date for this apprenticeship");
+                });
+
+                When(x => x.ChangeType == ChangeStatusType.Stop && x.StartDate < new DateTime(_currentDateTime.Now.Year, _currentDateTime.Now.Month, 1) && x.ChangeConfirmed != false, () =>
+                {
+                    RuleFor(r => r.DateOfChange)
+                               .Cascade(CascadeMode.StopOnFirstFailure)
```

Comment:
Can the second When be inside the first rather than duplicating most of the criteria?

Reply:
Done